Split keyboard support

sleepsounds opened the pull request:
Made it possible to use split keyboard supported on Sailfish OS 1.1.4.29.
Changed file made from 1.1.4.29 stock English keyboard so some other changes came from it.

sleepsounds commented:
Accidentally closed. So open again. Sorry.

BeholdMyGlory commented on src/ja_romaji.qml:
```diff
@@ -44,5 +77,10 @@ KeyboardLayout {
-    SpacebarRow { }
+    SpacebarRow {}
+
+    type: "custom"
```
From what I can tell, isn't Component.onCompleted unneeded here? applyAutocaps in KeyboardBase.qml will be called immediately on startup, and as long as `type` isn't empty, autocaps will automatically be set to false.

Also, I think it might be good to set `type` to a somewhat more descriptive name, e.g. "japan_romaji_anthy" or something like that.

sleepsounds replied:
For Component.onCompleted, I once tested deleting but it caused error. So I thought it's needed.
But I haven't examined much for why and I agree with you as look inside KeyboardBase.qml, it seems ok with only setting `type` to stop autocaps. So I have to test it more...

For the value of `type`, I think you are right.

BeholdMyGlory commented on src/ja_romaji.qml:
```diff
@@ -1,9 +1,38 @@
 import ".."
 
 KeyboardLayout {
+    splitSupported: true
```
This seems to break the candidate list when using a split keyboard. I think what's missing is a `verticalItem` in JaInputHandler.qml. Do you have the code for this already? If not I can probably whip something up using Xt9CpInputHandler.qml as a reference.

sleepsounds replied:
Oh I really forgot about this. So submitted it.

BeholdMyGlory commented:
OK, the code looks good to me, and it seems to work from what I've tested. I'll merge this once you've had a look at what I mentioned regarding autocaps.

sleepsounds commented:
Oh, well, I got it.
Moved `type` line to the first line of KeyboardLayout and it worked.
Soooo simple...Why haven't I tried...
Anyway, think it fixed! Also changed the value to "japan_romaji_anthy" as you suggested above.

BeholdMyGlory commented:
Awesome, I'll merge this then. Pull requestありがとうございました！

sleepsounds commented:
どういたしまして！
And thank you for update!
